tests/rust/parser_test.rs

```diff
@@ -1,10 +1,15 @@
 // Tests for the Rust parser
-struct Foo {
-    v: i32,
-}
+mod foo {
+    struct Foo {
+        v: i32,
+    }
 
-unsafe fn f(p: *mut Foo)
-//@ req [?q](*p).v |-> ?v;
-//@ ens [q](*p).v |-> v;
-{
+    //@ pred foo::Foo_own(t: thread_id_t, v: i32) = true;
```

Comment:
Sorry to be a nitpick, but I would prefer if you moved this definition out of the module; the natural behavior would be for this `foo::` prefix to be added automatically if the predicate is inside the `mod`. Moving it out of the `mod` makes this test case future-proof.

Reply:
Good point!